docs/Running-Tachyon-on-EC2-Yarn.md

```diff
@@ -146,52 +147,89 @@ master/workers, put `tachyon-site.properties` under `${TACHYON_HOME}/conf` on ea
+SLF4J: Class path contains multiple SLF4J bindings.
+SLF4J: Found binding in [jar:file:/hadoop/share/hadoop/common/lib/slf4j-log4j12-1.7.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
+SLF4J: Found binding in [jar:file:/tachyon/clients/client/target/tachyon-client-0.8.1-SNAPSHOT-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
+SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
+SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
+ApplicationMaster command: {{JAVA_HOME}}/bin/java -Xmx256M tachyon.yarn.ApplicationMaster 3 /tachyon localhost 1><LOG_DIR>/stdout 2><LOG_DIR>/stderr 
+Submitting application of id application_1445469376652_0002 to ResourceManager
+15/10/22 00:01:19 INFO impl.YarnClientImpl: Submitted application application_1445469376652_0002
+2015/10/22 00:01:29 Got application report from ASM for appId=2, clientToAMToken=null, appDiagnostics=, appMasterHost=, appQueue=default, appMasterRpcPort=0, appStartTime=1445472079196, yarnAppState=RUNNING, distributedFinalState=UNDEFINED, appTrackingUrl=http://TachyonMaster:8088/proxy/application_1445469376652_0002/A, appUser=ec2-user
+2015/10/22 00:01:39 Got application report from ASM for appId=2, clientToAMToken=null, appDiagnostics=, appMasterHost=, appQueue=default, appMasterRpcPort=0, appStartTime=1445472079196, yarnAppState=RUNNING, distributedFinalState=UNDEFINED, appTrackingUrl=http://TachyonMaster:8088/proxy/application_1445469376652_0002/A, appUser=ec2-user
+```
+
+From the output, we know the application ID to run Tachyon is 
+**`application_1445469376652_0002`**. This application ID is needed to kill the application.
+
+NOTE: currently Tachyon YARN framework does not guarantee to start the Tachyon master on the
+TachyonMaster machine; use the YARN Web UI to find out which machine is Tachyon master running on.
 
 # Test Tachyon
 
+When you know the IP of Tachyon master container, you can modify the `conf/tachyon-env.sh` to set
+ up environment variable `TACHYON_MASTER_ADDRESS` on each EC2 machine:
+
+```
+export TACHYON_MASTER_ADDRESS=Your_Master_IP
+```
+
 You can run tests against Tachyon to check its health:
 
 ```bash
 $ /tachyon/bin/tachyon runTests
 ```
 
-After the tests finish, visit Tachyon web UI at `http://{MASTER_IP}:19999` again. Click `Browse
+After the tests finish, visit Tachyon web UI at `http://{Your_Master_IP}:19999` again. Click `Browse
 File System` in the navigation bar, and you should see the files written to Tachyon by the above
 tests.
 
 
 # Stop Tachyon
 
-Tachyon can be stopped by using the following Yarn command where the application ID of Tachyon can
-be retrieved from either Yarn web UI or the output of `tachyon-yarn.sh`:
+Tachyon can be stopped by using the following YARN command where the application ID of Tachyon can
+be retrieved from either YARN web UI or the output of `tachyon-yarn.sh` as mentioned above. In 
```

Comment:
For consistency, it might make sense to use `APPLICATION_ID` instead of `application_1445469376652_0002`